### ISSUE TITLE
RCS1047 (remove Async suffix) is reported in an interface

### ISSUE THREAD

**Issue by R2D221** [JosefPihrt/Roslynator#665]
**Product and Version Used**: Roslynator VS extension, v2.9.0

**Steps to Reproduce**: Declare a new interface with async methods. The analyzer suggests to remove the Async suffix, even though the method is meant to be implemented asynchronously.

**Actual Behavior**:
```csharp
	public interface IAsyncReloadable
	{
		         //vvvvv-- RCS1047 is reported here
		Task ReloadAsync();
	}
```

**Expected Behavior**:
```csharp
	public interface IAsyncReloadable
	{
		Task ReloadAsync(); // It should not report anything
	}
```

I guess the analyzer sees that the `async` keyword isn't being used, but there's no way to use it on interfaces.

**Comment by R2D221** [comment]
Wait... I just noticed the Task wasn't resolving as System.Threading.Tasks.Task. I'm closing the issue because it was an error on my part.
